src/main/java/com/google/gwt/core/client/JavaScriptObject.java

```diff
@@ -144,7 +146,11 @@ public native String toSource() /*-{
    */
   @Override
   public final String toString() {
-    return JavaScriptObject.class.desiredAssertionStatus() ?
-        toStringVerbose(this) : toStringSimple(this);
-  }
+      if (true) {
+          return Document.Instance.toStringVerbose(this);
+      } else {
+          return JavaScriptObject.class.desiredAssertionStatus() ?
```

Comment:
It is your call.
The codes inside `if (false) {...}` is not called but is searchable by Eclipse References Workplace. sometime it is useful .